src/reformatters/common/zarr.py

```diff
@@ -35,7 +35,7 @@
 
 def get_zarr_store(dataset_id: str, version: str) -> zarr.storage.FsspecStore:
     if not Config.is_prod:
-        version = "dev"
+        version = "dev" if Config.is_dev else version
```

Comment:
I kept the `dev` behavior for consistency. It seemed a bit odd that many of our tests use `get_zarr_path(<dataset-id>, <some-version>)` but then the version is ignored, so I made this change to allow it to pass on through. Alternatively we could just set the version to `Config.env.value` (`dev` or `test`) but again callers passing in a version that is ignored is a bit awkward.